resources/assets/uad_lists.json

```diff
@@ -12890,11 +12890,11 @@
   {
     "id": "com.coloros.weather2",
     "list": "Oem",
-    "description": "colorOS weather app. Removal seems to trigger bootloops\nSee: https://github.com/0x192/universal-android-debloater/issues/211",
```

Comment:
There was a reason this package was in the `Unsafe` section. The exact issue is described in #158 (and mentioned in #211)

Reply:
Reading the last sentence on #211 he is talking about the Weather App (which shows up in the app drawer) whose package name is `com.coloros.weather2` not `com.coloros.weather.service`.
Maybe both got interchanged because of similar name.
Because weather service which is a background app is needed to show the weather in the quick setting panel, removing it causes the systemUI to crash.

Also I have tested on both oppo and realme (Android 11+) device and weather app can be removed without any issue.